### ISSUE TITLE
Make error strings more consistent 

### ISSUE THREAD

**Issue by lasers** [su8/pinky-bar#14]
Hi. I ran into minor inconsistency with py3status `pinkybar` module.

When error, display `pinkybar: unrecognized options: --su8`.   `NICE`
When error, display `pinkybar: program with --su8 and --lasers`. `NOT NICE`

The header `pinkybar:` comes from py3status's module name.
The message comes from `pinkybar` error message.

The problem is that when I get `unrecognized options`, I saw header and version too, `pinkybar: 1.00.00 `, so I skipped first two words in my code. The second message does not have header and version... is which why I skipped `recompile the` message. This should be:

When error, display `pinkybar: recompile the program with --su8 and --lasers`. `NICE`
(Real example, `--with-alsa or --with-oss`.)

See if we can make error strings "as_fn_error" more consistent. Keep in mind that the code to `skip first two words` can be removed too.

**Comment by su8** [comment]
```bash
# compiled --without-alsa
src/pinkybar --volume
```

output

>recompile the program --with-alsa or --with-oss

I don't see what's wrong with this.

**Comment by su8** [comment]
So you want the error message to start with `pinkybar: recompile the program --with-this-and-that` ?

**Comment by lasers** [comment]
For `py3status` pinkybar module, yes.

Print `pinkybar 1.00.00: recompile the program --with-alsa or --with-oss`.

This also make `pinkybar` error message looks more nice in terminal.

**Comment by su8** [comment]
And for the error messages that `--without-net` prints, e.g "Error: recompile the program --with-net", do I have to add `pinkybar:` infront too ?

**Comment by lasers** [comment]
Yes, probably. Everything should be consistent. `--bandwidth` and `--ipmac` prints `pinkybar 1.00:` okay, but not sure about `option`. So maybe other arguments are missing `option` too.... or not. Either way, everything still should be consistent.

**Comment by lasers** [comment]
Well, I realized this is more of a different message than others, so it make sense to leave it alone.

**Comment by su8** [comment]
Thanks for the suggestion, it should be fixed in the latest commit.

**Comment by lasers** [comment]
Missing version? Got `pinkybar: the program --with-alsa or --without oss`. If you prefer, I can skip only 1 word.

**Comment by su8** [comment]
Yes that would be great, because I might bump up the version in configure.ac and someone might ping me that pinkybar is using the old version.

**Comment by lasers** [comment]
Will the versions be removed in `--bandwidth`, `--ipmac`, etc too?

**Comment by su8** [comment]
That's handled by the library argp. Will bring the version back.
